### DISCUSSION TITLE
Mouse and Keyboard Controls

### DISCUSSION THREAD

**Discussion by UP778454** [AmyE123/TactiKit#29] [Map Editor Feedback]
I think the use of the right mouse button can be better served for resetting tiles rather than rotation, I would suggest using a key such a as R or Q to rotate something. Or possibly Q and E to rotate left or right.

**Comment by Lewis-Cooper101** [comment]
Yeah I think this would be helpful, like E for rotating clockwise and Q for anti-clockwise, so hover over the intended tile with the mouse and using the buttons to pick the direction you want. That way if you rotate past the orientation you want, you don't have to cycle all the way back around!
